components/cam/cime_config/config_component.xml

```diff
@@ -116,13 +116,13 @@
       <value compset="_CAM%ADIAB"     >-phys adiabatic</value>              
 
       <!-- SCREAM configurations -->
-      <value compset="_CAM5%SCREAM">-phys cam5 -shoc_sgs -microphys p3 -chem linoz_mam4_resus_mom_soag -rain_evap_to_coarse_aero -nlev 72 -rad rrtmgp</value>
+      <value compset="_CAM5%SCREAM-LR">-phys cam5 -shoc_sgs -microphys p3 -chem linoz_mam4_resus_mom_soag -rain_evap_to_coarse_aero -nlev 72 -rad rrtmgp</value>
```

Comment:
Should the low-res compset mimic the same prescribed aerosol configuration as the high res compset? Seems like having low res use prognostic and high res use prescribed makes for an apples to oranges comparison.

Reply:
Agreed. We should make low-res prescribed aerosol as well (based on a discussion with the exec council at the all-hands in Nov).